frontend/hooks/useBalanceContext.ts

```diff
@@ -39,7 +45,7 @@ export const useServiceBalances = (serviceConfigId: string) => {
   const serviceSafeBalances = useMemo<WalletBalanceResult[]>(
     () =>
       walletBalances?.filter((balance) =>
-        flatAddresses.includes(balance.walletAddress),
+        masterSafes.find(({address}) => balance.walletAddress === address),
```

Comment:
why to change, it seems masterSafes are in flatAddresses? 
<img width="401" alt="image" src="https://github.com/user-attachments/assets/c4374715-040f-4e7d-adcf-9859e02bf91a">

Reply:
hmm.. you're correct this is wrong :|

Reply:
will follow up with fixes, want to get this through so you can start working with the exports, assuming they are correct